core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.js

```diff
@@ -113,12 +106,14 @@ module.exports = class puppeteerUtilities {
    */
   async clickOn(selector) {
     try {
-      const [button] = await (this.page).$x(
-        '//' + '*[contains(text(), "' + selector + '")]');
+      const [button] = await this.page.$x(
+        /** Disabling the line because we need to use backticks for
```

Comment:
Don't understand this. You shouldn't be disabling eslint entirely anyway; only a specific pragma. What pragma is failing?

Reply:
only string must be in singlequotes is failing.

Reply:
fixed.. we don't need to disable any lint check here.